src/phase0/state_transition/mod.rs

```diff
@@ -70,6 +70,28 @@ pub enum InvalidOperation {
     IndexedAttestation(InvalidIndexedAttestation),
     #[error("invalid deposit: {0}")]
     Deposit(InvalidDeposit),
+    #[error("invalid block header: {0}")]
+    HeaderValidation(InvalidBeaconBlockHeader),
```

Comment:
i think we can simplify the name here:
```suggestion
    Header(InvalidBeaconBlockHeader),
```

Reply:
Simplified: 89fffa6